source/history.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var likely = require('./index.js');
+var utils = require('./utils');
+
+/**
+ * History
+ * @type {Object}
+ */
+var history = {
+    /**
+     * Called everytime the page url's been changed.
+     * Reinits all widgets with the new page url.
+     * @type {Function}
+     */
+    onUrlChanges: function () {
+        likely.initiate({
+            forceUpdate: true,
+            url: utils.getDefaultUrl(),
+        });
+    },
+    /**
+     * Inits pust/pop state events listeners
+     * @type {Function}
+     */
+    init: function () {
+        var that = this;
```

Comment:
Nit: it’s more a matter of taste, but I’d better use `.bind()` instead of `var that = this`. It looks like [there’s no general preference](http://codereview.stackexchange.com/questions/49872/using-var-self-this-or-bindthis) for either way and both are used equally, but I prefer `.bind()` more as it’s the newer approach to solving the problem.